pallets/automation-time/src/lib.rs

```diff
@@ -508,6 +576,49 @@ pub mod pallet {
 			Ok(().into())
 		}
 
+		/// Schedule a task to increase delegation to a specified up to a minimum balance
+		/// Task will reschedule itself to run on a given frequency until a failure occurs
+		///
+		/// # Parameters
+		/// * `execution_time`: The unix timestamp when the task should run for the first time
+		/// * `frequency`: Number of seconds to wait inbetween task executions
+		/// * `collator_id`: Account ID of the target collator
+		/// * `account_minimum`: The minimum amount of funds that should be left in the wallet
```

Comment:
Good concern, but I think we're safe since these funds aren't getting spent, just reserved. So the account doesn't get reaped. I couldn't find documentation to back me up, but I managed to test it locally-
![image](https://user-images.githubusercontent.com/16072671/179640663-5d20e86e-e267-469e-830f-18a46d712030.png)

Reply:
Also verified that rewards still get paid out when you have 0 transferrable balance- 
![image](https://user-images.githubusercontent.com/16072671/179641770-0e5dc24d-7aa3-4ddb-8726-844917980f54.png)